pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -13,10 +14,27 @@ self: super:
 
 let
   inherit (neovimUtils) grammarToPlugin;
-  generatedGrammars = callPackage ./generated.nix {
-    inherit (tree-sitter) buildGrammar;
+
+  overrides = prev: {
+    ocamllex =
+      if stdenv.hostPlatform.isDarwin then
+        (prev.ocamllex.overrideAttrs {
+          src = prev.ocamllex.src.override {
+            hash = "sha256-UBGVc98lrtTCp/kYDEFM/8iG9n7Tekx+xbE7Wdyp2uQ=";
+          };
+        })
+      else
+        prev.ocamllex;
   };
```

Comment:
Feels like a maintenance nightmare.. Can we add a TODO: to remove after https://github.com/314eter/tree-sitter-ocamllex/issues/10 is resolved

Reply:
> Feels like a maintenance nightmare..

Yes, it is unfortunately

> after https://github.com/314eter/tree-sitter-ocamllex/issues/10 is resolved

I suspect the grammar is not actively maintained

> Is this observable on hydra? Or is it just when users try to build natively before it's cached from hydra?

I don't know how to filter builds by package, except for this:
```
perchun@darwin01 ~/nixpkgs (master)> nix-build -A vimPlugins.nvim-treesitter-parsers.ocamllex.origGrammar.src
this path will be fetched (0.04 MiB download, 0.20 MiB unpacked):
  /nix/store/fbrpa3pg2c4ricpzw8nc50zwrc0wf3hr-source
copying path '/nix/store/fbrpa3pg2c4ricpzw8nc50zwrc0wf3hr-source' from 'https://nix-community.cachix.org'...
/nix/store/fbrpa3pg2c4ricpzw8nc50zwrc0wf3hr-source
```